test/integration/per-node-stream-metrics.test.js

```diff
@@ -29,8 +29,10 @@ describe('metricsStream', () => {
             wsPort: wsPort1,
```

Comment:
In `beforeEach` create a new stream. And then take it's id and put it in the config of broker1, i.e. `streamr.streamId` and then subscribe to that stream in the actual test case.